### PR TITLE
Accepting ceph port as parameter

### PR DESCRIPTION
**Pull request by tusharuipath**
1. making ceph port as parameter

### CODE REVIEW

File: orchestrator/orchestratorAutomation.ps1 [hunk 1]
```diff
@@ -26,6 +26,8 @@ Param (
    [string] $aifip,
    [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
    [string] $aifport
+   [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
+   [string] $cephport
```
Comment (snvenkat) [inline orchestrator/orchestratorAutomation.ps1:30]: We should name it storage port as it could be anything tomorrow.

Comment (tusharuipath) [reply orchestrator/orchestratorAutomation.ps1]: Done

File: orchestrator/orchestratorAutomation.ps1 [hunk 2]
```diff
@@ -54,7 +60,7 @@ $STATIC_NODES_STRING='
     <add key="AiFabric.Logs" value="false" />
     <add key="AiFabric.ModuleEnabled" value="true" />
     <add key="AiFabric.FeatureEnabledByDefault" value="true" />
-    <add key="AiFabric.ModelStorageUrl" value="https://{{aifip}}:31443" />
+    <add key="AiFabric.ModelStorageUrl" value="https://{{aifip}}:{{storageport}}" />
     <add key="AiFabric.MLPackagingInstructionsUrl" value="something" />
```
Comment (mohitseth-nitrr) [inline orchestrator/orchestratorAutomation.ps1:64]: completely unrelated ... but can you set packaging instructionUrl value to 
https://docs.uipath.com/ai-fabric/docs/building-ml-packages

Comment (tusharuipath) [reply orchestrator/orchestratorAutomation.ps1]: Done

**Pull request merged**